src/native/eventpipe/ds-dump-protocol.c

```diff
@@ -115,9 +180,17 @@ dump_protocol_helper_generate_core_dump (
 		ep_raise_error ();
 	}
 
-	ipc_result = ds_rt_generate_core_dump (commandId, payload);
+    ep_char8_t errorMessage[1024];
```

Comment:
Is this large enough for all error data? I recall the error messages being quite long but I might be confusing it with the verbose logging.

Reply:
The error messages really should not be longer than that (createdump logging could be but that doesn't go through here) and if they were it would just truncate them.  I was wondering if I should allocate this on the heap or stack since it is this big.

Reply:
The frequency that this gets called doesn't have me worried about the size too much.